.github/workflows/test-ng-filtering.yml

```
@@ -0,0 +1,70 @@
+        description: 'Enable debug mode'
+        required: false
+        default: 'true'
+        type: choice
+        options:
+          - 'true'
+          - 'false'
+
+jobs:
+  test-ng-fetch:
+    runs-on: ubuntu-latest
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)